src/elements/content-sidebar/__tests__/withSidebarAnnotations.test.js

```diff
@@ -249,6 +265,75 @@ describe('elements/content-sidebar/withSidebarAnnotations', () => {
         );
     });
 
+    describe('updateAnnotation()', () => {
+        const sidebarPanelsRef = { refresh: jest.fn() };
+
+        test.each`
+            action                 | expectedIsPending
+            ${Action.UPDATE_START} | ${true}
+            ${Action.UPDATE_END}   | ${false}
+        `(
+            'should update appropriate annotation in the feed given action = $action',
+            ({ action, expectedIsPending }) => {
+                const annotation = { id: '123', status: 'resolved' };
+                const annotatorStateMock = {
+                    annotation,
+                    action,
+                };
+
+                const wrapper = getWrapper({ annotatorState: annotatorStateMock });
+                const instance = wrapper.instance();
+
+                instance.updateAnnotation();
+
+                const expectedAnnotationData = { ...annotation, isPending: expectedIsPending };
+                expect(feedAPI.updateFeedItem).toBeCalledWith(expectedAnnotationData, annotation.id);
+            },
+        );
+
+        test.each`
```

Comment:
If we can avoid duplicating the logic, we can avoid duplicating the tests.

Reply:
Right, once I've extracted the logic for refreshing Activity Sidebar, there is just one instance of that test.